src/test/resources/jsonpatch/extended/omit.json

```diff
@@ -0,0 +1,41 @@
+{
+    "errors": [
```

Comment:
Some test cases on:
1. path "" with matching value?
1. path "" with simple values 7 and "abc"
2. path / with simple values